vcd/resource_vcd_vapp_vm.go

```diff
@@ -364,15 +414,15 @@ func resourceVcdVAppVmCreate(d *schema.ResourceData, meta interface{}) error {
 		}
 	}
 
-	d.SetId(d.Get("name").(string))
+	d.SetId(vm.VM.ID)
```

Comment:
How does the "upgrade" from 2.5.0 looks like for someone who already has VMs? Are there any challenges?

Reply:
I haven't observed any while I was trying this out. Tried 2.5 -> 2.6 multiple times and `apply` was clean.

Reply:
I have seen no issues so far.